### ISSUE TITLE
[FR] Show commands in REPL when Ctrl+Enter

### ISSUE THREAD

**Issue by jmboehm** [JunoLab/Juno.jl#285]
Under the "UI options/Result Display Mode" it would be great to have another option: "Display code and results in the console" which would send (and display) the executed code and any following output line-by-line to the REPL window.

The advantage would be:
- it's easier to see which output corresponds to which line of code
- it makes it easier to copy-paste the REPL output into a "log" of what you've run during a session.

**Comment by dlill** [comment]
Yes, and it would be great if it is possible to add the executed commands to the history which can be accessed in the REPL by pressing ArrowUp!
